README.md

```diff
@@ -21,7 +19,7 @@ The apt module automates obtaining and installing software packages on \*nix sys
 * System repositories
 * Authentication keys
 
-**Note**: Setting the apt module's `purge_sources_list` and `purge_sources_list_d` parameters to 'true' will **destroy** any existing content that was not declared with Puppet. The default for these parameters is 'false'.
+**Note**: Setting the apt module's `purge['sources.list']` and `purge['sources.list.d']` parameters to `true` will **destroy** any existing content that was not declared with Puppet. The default for these parameters is `false`.
```

Comment:
Actually, the default is `true`: https://github.com/puppetlabs/puppetlabs-apt/blob/next/manifests/params.pp#L71-L76 :smile:.